src/main/java/javax/resource/spi/Activation.java

```diff
@@ -54,7 +53,7 @@
 	/**
 	 * Indicates the message listener type(s) associated with this activation.
 	 * 
-	 * @return The Java types of the Message Listener interface this
+	 * @return The Java&trade; types of the Message Listener interface this
```

Comment:
A lot of gratuitous introductions of "&trade;".  Doing it once at the top level package javadocs would be more than enough.

Reply:
I think this is the first reference to Java but removed other additional tm references throughout